edgecontext/validator.go

```diff
@@ -43,22 +49,21 @@ func shouldShortCircutError(err error) bool {
 // ValidateToken parses and validates a jwt token, and return the decoded
 // AuthenticationToken.
 func ValidateToken(token string) (*AuthenticationToken, error) {
-	sec, err := store.GetVersionedSecret(authenticationPubKeySecretPath)
-	if err != nil {
-		return nil, err
+	keys, ok := keysValue.Load().(keysType)
+	if !ok {
+		// This would only happen when all previous middleware parsing failed.
+		return nil, errors.New("no public keys loaded")
 	}
 
-	// TODO 1: Patch upstream to support key rotation natively:
+	// TODO: Patch upstream to support key rotation natively:
```

Comment:
Nice, can we add an issue in our repository to keep track of this?

Reply:
https://github.com/reddit/baseplate.go/issues/73